### PR TITLE
fix: fix validation issues

### LINKED ISSUE #30: [Auto] AI Gallery Standard Validation
# AI Gallery Standard Validation: FAILED

## Repository Management:
- [x] <b>README.md File</b>.
- [x] <b>LICENSE.md File</b>.
- [x] <b>SECURITY.md File</b>.
- [x] <b>CODE_OF_CONDUCT.md File</b>.
- [x] <b>CONTRIBUTING.md File</b>.
- [x] <b>ISSUE_TEMPLATE.md File</b>.
- [ ] :warning: <b>Topics on repo contains ['azd-templates', 'ai-azd-templates']</b>. <a href=https://aka.ms/ai-template-standards>[How to fix?]</a>

  - [ ] :warning: Error: azd-templates is missing in topics.
  - [ ] :warning: Error: ai-azd-templates is missing in topics.

## Source code structure and conventions:
- [x] <b>azure-dev.yml File</b>.
- [x] <b>azure.yml File</b>.
- [x] <b>infra Folder</b>.
- [x] <b>.devcontainer Folder</b>.

## Functional Requirements:
- [x] <b>azd up</b>.
- [x] <b>azd down</b>.

### PR DESCRIPTION
**Pull request by anfibiacreativa**
Fix validation closes #30 
Pending topics
Pending to review validator warnings for `azd up` for maintainer, for security conformance.

@cassiebreviu @kristenwomack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,15 +151,26 @@ To clean up all the resources created by this sample:
 3. When asked if you want to permanently delete the resources, enter `y`
 
 The resource group and all the resources will be deleted.
-## Costs
+
+## Guidance
+
+### Region Availability
+
+This template uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
```
Comment (anfibiacreativa) [inline README.md:159]: @cassiebreviu  please update this generic section with actual models and region information before merge.

Comment (cassiebreviu) [reply README.md:159]: @anfibiacreativa you can find this information in the infrastructure. Let me know once you have updated it and I will merge.

File: README.md [hunk 2]
```diff
@@ -151,15 +151,26 @@ To clean up all the resources created by this sample:
...
+
+### Costs
+
 You can estimate the cost of this project's architecture with [Azure's pricing calculator](https://azure.microsoft.com/pricing/calculator/)
- 
-- Azure OpenAI: Standard tier, GPT and Ada models. Pricing per 1K tokens used, and at least 1K tokens are used per question. [Pricing](https://azure.microsoft.com/pricing/details/cognitive-services/openai-service/)
-- Azure AI Speech: Pay as you go, Standard,	$1 per hour [Pricing](https://azure.microsoft.com/en-gb/pricing/details/cognitive-services/speech-services/)
 
-## Securtiy Guidelines
+* [Azure Product] - [plan type] [link to pricing for product](https://azure.microsoft.com/pricing/)
```
[the remark this answers is not in the document]
Comment (cassiebreviu) [reply README.md:166]: Here is the pricing link for you: https://azure.microsoft.com/en-us/pricing/details/ai-studio/

File: README.md [hunk 3]
```diff
@@ -151,15 +151,26 @@ To clean up all the resources created by this sample:
...
 
 The resource group and all the resources will be deleted.
-## Costs
+
+## Guidance
+
+### Region Availability
+
+This template uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
+  * We recommend using [SUGGESTED REGION]
```
[the remark this answers is not in the document]
Comment (cassiebreviu) [reply README.md:160]: Please update this to refer them to check this list: https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models

**Pull request closed without merging**